docs/development.md

```diff
@@ -0,0 +1,108 @@
+# Development
+## Running locally
+
+### Major Dependencies
+
+- [Node.js](https://nodejs.org/en/) (>= 14)
+- [NPM](https://www.npmjs.com/) (>= 6.14)
+- [Redis](https://redis.io/) (>= 5.0.5)
+
+### Installation
+
+1. Clone repository and change directory:
+
+   ```
+   git clone https://github.com/ministryofjustice/hmpps-book-secure-move-frontend && cd hmpps-book-secure-move-frontend
+   ```
+
+1. Install node dependencies:
+
+   ```
+   npm install
+   ```
+
+1. Build the assets
+
+    ```
+    npm run build
+    ```
+
+## Configuring the application
+
+Create a copy of the example environment variable file and add values for the keys:
+
+   ```
+   cp .env.example .env
+   ```
+
+Set the [environment variables](#environment-variables) accordingly.
+
+## Running the application
+
+### In production mode
```

Comment:
This feels like a duplication of the repo readme now. What's the thought behind moving to a separate piece of documentation?

Reply:
*nods*

This is another pattern I borrowed from `alphagov/govuk-frontend`. They have a small `Readme.md`, and then a big contributing documentation detailing how everything works. I felt the folder was a little overboard for our project at the moment, however I did like the streamlined main `Readme.md` with any detailed information in separate documents.

So that ends up with:
- `Readme.md` - modified
- `CONTRIBTING.md` - standard addition
- `CODE_OF_CONDUCT` - standard addition
- `docs/projectstructure.md` - renamed previous docs

- `docs/development.md`- moved from `Readme.md`
- `docs/testing.md`- moved from `Readme.md`
- `docs/releasing.md`- moved from `Readme.md`
- `docs/coding-standards.md`- moved from `Readme.md`
- `docs/environment-variables.md`- moved from `Readme.md`

Personally, I think moving the development specific docs out of the `Readmemd` makes both easier to read and find, and I think the same applies on separate parts of development.  However, I'm not convinced by their approach of having separate files for css and js coding standards - I think that's a step too far for our usage.